conf.d/done.fish

```diff
@@ -111,17 +111,29 @@ end
 function __done_is_tmux_window_active
     set -q fish_pid; or set -l fish_pid %self
 
-    # find the outermost process within tmux
-    # ppid != "tmux" -> pid = ppid
-    # ppid == "tmux" -> break
+    # Find the tmux PID that is the parent of the fish process
     set tmux_fish_pid $fish_pid
-    while set tmux_fish_ppid (ps -o ppid= -p $tmux_fish_pid | string trim)
-        and ! string match -q "tmux*" (basename (ps -o command= -p $tmux_fish_ppid))
+    while true
+        if test -e /proc/$tmux_fish_pid/status
+            # awk is used instead of ps with -o ppid= because the latter is not
+            # supported by busybox
+            set tmux_fish_ppid (awk '/PPid:/ {print $2}' /proc/$tmux_fish_pid/status)
+            set tmux_fish_ppid_cmd_full (busybox ps | awk "{if (\$1 == $tmux_fish_ppid) print \$NF}")
```

Comment:
@154pinkchairs you are using busybox in this line, but as far as I understand it is not available in every system. What's the idea?

Reply:
hmm yup, my idea was just to have the script work with both, must've missed it

Reply:
Would you like to work on alternative. The existing function is error prone and I will have to choose one of the proposed options. This doesn't sound bad if it also covers the other edge cases.